requirements.txt

```diff
@@ -7,8 +7,10 @@ Flask-SQLAlchemy==2.4.1
 Flask-Mail-SendGrid==0.3
 Flask-SocketIO==5.0.1
 Flask-Cors==3.0.10
+Flask-Limiter==1.4
 email-validator==1.1.2
-Werkzeug==1.0.1
+Werkzeug==2.0.1
```

Comment:
should be able to remove this as flask implicitly imports it

Reply:
deleted